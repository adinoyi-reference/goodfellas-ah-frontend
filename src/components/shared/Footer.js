import React from 'react';

const Footer = () => (
  <footer>
    <div className="copyright">
      <p>&copy;Goodfellas</p>
    </div>
  </footer>
);

export default Footer;
