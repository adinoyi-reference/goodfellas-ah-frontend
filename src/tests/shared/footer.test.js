
import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { MemoryRouter } from 'react-router-dom';
import Footer from '../../components/shared/Footer';
import Root from '../../root';

let wrapped;

beforeEach(() => {
  wrapped = mount(

    <Root>
      <MemoryRouter initialEntries={[{ pathname: '/', key: 'testKey' }]}>
        <Footer />
      </MemoryRouter>
    </Root>,
  );
});

afterEach(() => wrapped.unmount());
describe('Loading', () => {
  describe('render', () => {
    test('container should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
