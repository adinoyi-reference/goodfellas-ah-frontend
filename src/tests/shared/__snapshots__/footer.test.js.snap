// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render container should render as expected 1`] = `
<_default>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <MemoryRouter
      initialEntries={
        Array [
          Object {
            "key": "testKey",
            "pathname": "/",
          },
        ]
      }
    >
      <Router
        history={
          Object {
            "action": "POP",
            "block": [Function],
            "canGo": [Function],
            "createHref": [Function],
            "entries": Array [
              Object {
                "hash": "",
                "key": "testKey",
                "pathname": "/",
                "search": "",
              },
            ],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "index": 0,
            "length": 1,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "key": "testKey",
              "pathname": "/",
              "search": "",
            },
            "push": [Function],
            "replace": [Function],
          }
        }
      >
        <Footer>
          <footer>
            <div
              className="footer-wrapper"
            >
              <Link
                className="col-lg-3"
                replace={false}
                to="/"
              >
                <a
                  className="col-lg-3"
                  href="/"
                  onClick={[Function]}
                >
                  <p>
                    About
                  </p>
                </a>
              </Link>
              <Link
                className="col-lg-3"
                replace={false}
                to="/"
              >
                <a
                  className="col-lg-3"
                  href="/"
                  onClick={[Function]}
                >
                  <p>
                    Contact
                  </p>
                </a>
              </Link>
              <Link
                className="col-lg-3"
                replace={false}
                to="/"
              >
                <a
                  className="col-lg-3"
                  href="/"
                  onClick={[Function]}
                >
                  <p>
                    Terms
                  </p>
                </a>
              </Link>
              <Link
                className="col-lg-3"
                replace={false}
                to="/"
              >
                <a
                  className="col-lg-3"
                  href="/"
                  onClick={[Function]}
                >
                  <p>
                    Privacy
                  </p>
                </a>
              </Link>
            </div>
          </footer>
        </Footer>
      </Router>
    </MemoryRouter>
  </Provider>
</_default>
`;
